x-pack/solutions/security/plugins/security_solution/public/entity_analytics/components/privileged_user_monitoring/components/privileged_access_detection/pad_chart/pad_user_name_list.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { useExpandableFlyoutApi } from '@kbn/expandable-flyout';
+import { EuiFlexGroup, EuiFlexItem, EuiLink, EuiText } from '@elastic/eui';
+import {
+  EntityPanelKeyByType,
+  EntityPanelParamByType,
+} from '../../../../../../flyout/entity_details/shared/constants';
+import { padChartStyling } from '.';
+
+export const UserNameList: React.FC<{ userNames: string[] }> = ({ userNames }) => {
+  const { openRightPanel } = useExpandableFlyoutApi();
+
+  const openUserFlyout = (userName: string) => {
+    const id = EntityPanelKeyByType.user;
+
+    if (id) {
+      openRightPanel({
+        id,
+        params: {
+          [EntityPanelParamByType.user ?? '']: userName,
```

Comment:
`[EntityPanelParamByType.user ?? '']` when will EntityPanelParamByType.user not be present and what is the effect of setting the empty key here?